packages/web-twig/src/Resources/components/CheckboxField/README.md

```diff
@@ -29,27 +29,27 @@ Without lexer:
-| ----------------- | ----------------------------- | ------- | -------- | ------------------------------ |
-| `class`           | `string`                      | `null`  | no       | Custom CSS class               |
-| `id`              | `string`                      | `null`  | no       | Input and label identification |
-| `isChecked`       | `bool`                        | `false` | no       | If true, input is checked      |
-| `isDisabled`      | `bool`                        | `false` | no       | If true, input is disabled     |
-| `isItem`          | `bool`                        | `false` | no       | To render in [Item][item] mode |
-| `isLabelHidden`   | `bool`                        | `false` | no       | If true, label is hidden       |
-| `isRequired`      | `bool`                        | `false` | no       | If true, input is required     |
-| `label`           | `string`                      | —       | yes      | Label text                     |
-| `message`         | `string`                      | `null`  | no       | Validation or help message     |
-| `name`            | `string`                      | `null`  | no       | Input name                     |
-| `validationState` | `success`, `warning`, `error` | `null`  | no       | Type of validation state       |
-| `value`           | `string`                      | `null`  | no       | Input value                    |
+| Prop name         | Type                                    | Default | Required | Description                    |
+| ----------------- | --------------------------------------- | ------- | -------- | ------------------------------ |
+| `class`           | `string`                                | `null`  | no       | Custom CSS class               |
+| `id`              | `string`                                | `null`  | no       | Input and label identification |
+| `isChecked`       | `bool`                                  | `false` | no       | If true, input is checked      |
+| `isDisabled`      | `bool`                                  | `false` | no       | If true, input is disabled     |
+| `isItem`          | `bool`                                  | `false` | no       | To render in [Item][item] mode |
+| `isLabelHidden`   | `bool`                                  | `false` | no       | If true, label is hidden       |
+| `isRequired`      | `bool`                                  | `false` | no       | If true, input is required     |
+| `label`           | `string`                                | —       | yes      | Label text                     |
+| `message`         | `string`                                | `null`  | no       | Validation or help message     |
+| `name`            | `string`                                | `null`  | no       | Input name                     |
+| `validationState` | `success`, `warning`, `danger`, `error` | `null`  | no       | Type of validation state       |
```

Comment:
again please link to docs/dictionariesmd and deprecation. See other components, where dictionaries are already implemented.

Reply:
done